osmnx/downloader.py

```diff
@@ -275,7 +278,7 @@ def _get_pause(base_endpoint, recursive_delay=5, default_duration=60):
 
     try:
         url = base_endpoint.rstrip("/") + "/status"
-        response = settings.session.get(url, headers=_get_http_headers())
+        response = session.get(url, headers=_get_http_headers())
```

Comment:
`_create_session` is not called in the `_get_pause` function, but it is called in the other functions that make HTTP requests to the Overpass server?